admin_guide/image_signatures.adoc

```diff
@@ -124,9 +124,15 @@ $ oc adm policy add-cluster-role-to-user system:image-auditor <user_name>
 endif::[]
 ====
 
+[IMPORTANT]
+====
 Using the `--save` flag on already verified image together with invalid GPG key
-or invalid expected identity causes the saved verification status to be removed,
-and the image will become unverified.
+or invalid expected identity causes the saved verification status with all
+previous signatures to be removed, and the image will become unverified. 
```

Comment:
s/previous// ?

Reply:
couldn't come up with any better adjective so I just removed the 'previous'

Reply:
sorry if it wasn't clear, that's exactly what i was suggesting you do :)